test/v8-updates/test-trace-gc-flag.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+// This test verifies that `--trace-gc` flag is well integrated.
+// We'll check here, that the console outputs gc events properly.
+require('../common');
+
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+
+const fixtures = require('../common/fixtures');
+
+{
+  const childProcess = spawnSync(process.execPath, [
+    '--trace-gc',
+    '--expose-gc',
+    fixtures.path('gc.js'),
+  ]);
+  const output = childProcess.stdout.toString();
+  const lines = splitByLine(output);
+
+  const scavengeRegex = /\bScavenge\b/;
+  const expectedOutput = [
+    scavengeRegex,
+    scavengeRegex,
+    scavengeRegex,
+    scavengeRegex,
+    /\bMark-sweep\b/,
+  ];
+  lines.forEach((line, index) => {
+    const expectedRegex = new RegExp(expectedOutput[index]);
+    assert.match(line, expectedRegex);
```

Comment:
```suggestion
    assert.match(line, expectedOutput[index]);
```

`expectedOutput` already contains regexes.

Reply:
In fact, I did like it first, then I got an error but I misread it.

I finally trim the output string and it worked ^^

Thanks a lot 🙏